src/createLambdaProxyEvent.js

```diff
@@ -75,6 +75,15 @@ module.exports = function createLambdaProxyEvent(
   const pathParams = { ...request.params };
 
   let token = headers.Authorization || headers.authorization;
+  let headerAuthorizer;
+  try {
+    const headerAuthorizerString = headers.Authorizer || headers.authorizer;
+    headerAuthorizer = JSON.parse(headerAuthorizerString);
```

Comment:
Opps, I'll put an `if (headerAuthorizerString) {}` around this so that we won't log errors when the authorizer header isn't set.